src/lib/storage/fixed_size_attribute_vector.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <vector>
+#include "storage/base_attribute_vector.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+template <typename T>
+class FixedSizeAttributeVector : public BaseAttributeVector {
+ public:
+  explicit FixedSizeAttributeVector(std::vector<T> value_ids) : _value_ids{value_ids} {}
```

Comment:
In our team, we decided that the std::vector used internally should be hidden from the interface so exchanging the implementation is easier -- so we decided to just pass a size and then use `set` from there on.

Reply:
valid critics but we like our move constructor for inserting data